### PR TITLE
Release version 12.12.0

### PR DESCRIPTION
**Pull request by stevendanna**
Signed-off-by: Steven Danna <steve@chef.io>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,93 @@
 # Chef Server Changelog
 
+# Change Log
```
Comment (stevendanna) [inline CHANGELOG.md]: Remove this, bad paste

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,93 @@
...
+* Shadae Holmes
+* Bryan McLellan
+* Nathan L Smith
+* Ryan Cragun
+* Sean Nolen
+
+**Fixed bugs:**
+
+- chef-server-ctl password command does not accept special characters  [\#366](https://github.com/chef/chef-server/issues/366)
+- `chef-server-ctl cleanse` removes plugin config, breaking reconfigure [\#115](https://github.com/chef/chef-server/issues/115)
```
Comment (stevendanna) [inline CHANGELOG.md]: Remove, this was fixed a long time ago.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,93 @@
...
+- Update docs for development environment [\#1027](https://github.com/chef/chef-server/pull/1027) ([thommay](https://github.com/thommay))
+- Move to the latest omnibus-software [\#1026](https://github.com/chef/chef-server/pull/1026) ([stevendanna](https://github.com/stevendanna))
+- Retry connection errors when bootstrapping bifrost [\#1025](https://github.com/chef/chef-server/pull/1025) ([ryancragun](https://github.com/ryancragun))
+- Bump version for development [\#1019](https://github.com/chef/chef-server/pull/1019) ([stevendanna](https://github.com/stevendanna))
+- Release note version update [\#1011](https://github.com/chef/chef-server/pull/1011) ([tas50](https://github.com/tas50))
+- Accept passwords with special chars in `chef-server-ctl password` [\#1005](https://github.com/chef/chef-server/pull/1005) ([stevendanna](https://github.com/stevendanna))
+- Validate user's 'name' before 'username' [\#1004](https://github.com/chef/chef-server/pull/1004) ([stevendanna](https://github.com/stevendanna))
+- Added email host and from address configuration for oc\_id [\#553](https://github.com/chef/chef-server/pull/553) ([dissonanz](https://github.com/dissonanz))
+
+
```
Comment (stevendanna) [inline CHANGELOG.md]: extra newline here

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -8,6 +8,42 @@ This document contains release notes for the current major release and all patch
...
+- `chef-server-ctl backup` correctly backs up configuration
+
+Starting in version 12.10.0, a bug in the backup command produced
+backups that did not include the configuration data in the resulting
+tarball. This bug is now resolved. We recommend taking a new backup
+after upgrading to 12.12.0.
+
+- Search respects rows parameter when using ElasticSearch
+
+When using ElasticSearch as search index, the Chef Server now
```
Comment (stevendanna) [inline RELEASE_NOTES.md]: `When configured to use ElasticSearch`

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,93 @@
...
+**Merged pull requests:**
+
+- Bump omnibus-software to roll back to runit 2.1.1 [\#1063](https://github.com/chef/chef-server/pull/1063) ([stevendanna](https://github.com/stevendanna))
+- Minor README changes [\#1062](https://github.com/chef/chef-server/pull/1062) ([stevendanna](https://github.com/stevendanna))
+- Update omnibus-software to get the latest server-jre [\#1059](https://github.com/chef/chef-server/pull/1059) ([stevendanna](https://github.com/stevendanna))
+- Fix the fips build [\#1056](https://github.com/chef/chef-server/pull/1056) ([stevendanna](https://github.com/stevendanna))
+- Remove coverdata files from oc\_erchef dir [\#1055](https://github.com/chef/chef-server/pull/1055) ([stevendanna](https://github.com/stevendanna))
+- Fix Makefiles to not distclean by default [\#1054](https://github.com/chef/chef-server/pull/1054) ([stevendanna](https://github.com/stevendanna))
+- Update depselector\_rb Gemfile to use HTTPS [\#1053](https://github.com/chef/chef-server/pull/1053) ([stevendanna](https://github.com/stevendanna))
+- Revert ruby to 2.2.6 [\#1052](https://github.com/chef/chef-server/pull/1052) ([stevendanna](https://github.com/stevendanna))
```
Comment (marcparadise) [inline CHANGELOG.md]: May be worth editing out this one and the upgrade we reverted from, to keep it easier to scan

File: RELEASE_NOTES.md [hunk 6]
```diff
@@ -8,6 +8,42 @@ This document contains release notes for the current major release and all patch
...
+correctly respects the `rows` parameter to search requests rather than
+returning all rows.
+
+We recommend upgrading to 12.12.0 for all users of Chef Backend, which
```
Comment (marcparadise) [inline RELEASE_NOTES.md]: s/which uses ElasticSearch.*$/to receive the search fix

File: RELEASE_NOTES.md [hunk 7]
```diff
@@ -8,6 +8,42 @@ This document contains release notes for the current major release and all patch
...
+tarball. This bug is now resolved. We recommend taking a new backup
+after upgrading to 12.12.0.
+
+- Search respects rows parameter when using ElasticSearch
+
+When configured to use ElasticSearch, the Chef Server now correctly
+respects the `rows` parameter to search requests rather than returning
+all rows.
+
+We recommend upgrading to 12.12.0 for all users of Chef Backend, which
```
Comment (marcparadise) [inline RELEASE_NOTES.md]: s/, which uses.*/to receive this search fix.

(Second time adding this. The version in the review itself is missing...)

**Pull request merged**